packages/framework/ini-files/config-specs.ini

```
@@ -1815,6 +1899,54 @@ opt-set-cmake-var Trilinos_ENABLE_TrilinosBuildStats     BOOL FORCE : OFF
 
 use RHEL8_POST
 
+[rhel8_aue-gcc-openmpi_release_static_no-kokkos-arch_no-asan_complex_no-fpic_mpi_no-pt_no-rdc_no-uvm_deprecated-on_compsim]
+use COMPILER|GNU
+use BUILD-TYPE|RELEASE
+use NODE-TYPE|SERIAL
+# Shared libraries until static ones are available in AUE container
+opt-set-cmake-var BUILD_SHARED_LIBS      BOOL   : ON
+opt-set-cmake-var TPL_FIND_SHARED_LIBS   BOOL   : ON
+use KOKKOS-ARCH|NO-KOKKOS-ARCH
+use USE-ASAN|NO
+use USE-COMPLEX|YES
+use USE-FPIC|NO
+use USE-MPI|YES
+use USE-PT|NO
+use USE-RDC|NO
+use USE-UVM|NO
+use USE-DEPRECATED|YES
+use PACKAGE-ENABLES|COMPSIM
+
+opt-set-cmake-var Trilinos_ENABLE_ALL_FORWARD_DEP_PACKAGES BOOL : OFF
+opt-set-cmake-var Trilinos_ENABLE_ALL_OPTIONAL_PACKAGES BOOL : OFF
+opt-set-cmake-var Trilinos_ASSERT_MISSING_PACKAGES BOOL : OFF
+opt-set-cmake-var Trilinos_ENABLE_FLOAT BOOL FORCE : ON
+opt-set-cmake-var Trilinos_ENABLE_COMPLEX_FLOAT BOOL : OFF
+opt-set-cmake-var Trilinos_SHOW_DEPRECATED_WARNINGS BOOL : OFF
+
+opt-set-cmake-var TPL_ENABLE_BLAS BOOL : ON
+opt-set-cmake-var TPL_ENABLE_Boost BOOL : ON
+opt-set-cmake-var TPL_ENABLE_LAPACK BOOL : ON
+opt-set-cmake-var TPL_ENABLE_METIS BOOL : ON
+opt-set-cmake-var TPL_ENABLE_ParMETIS BOOL : ON
+opt-set-cmake-var TPL_ENABLE_Scotch BOOL : ON
+opt-set-cmake-var TPL_ENABLE_UMFPACK BOOL : ON
+# Disabled until easily distributed y12m is available
+opt-set-cmake-var TPL_ENABLE_y12m BOOL : OFF
+
+opt-set-cmake-var TPL_UMFPACK_LIBRARIES STRING FORCE : ${UMFPACK_LIB|ENV}/libumfpack.so
+opt-set-cmake-var TPL_UMFPACK_INCLUDE_DIRS PATH FORCE : ${UMFPACK_INC|ENV}
+
+opt-set-cmake-var CMAKE_GENERATOR STRING : Ninja
+opt-set-cmake-var CMAKE_CXX_FLAGS STRING : -O2 -finline-functions -w
+opt-set-cmake-var CMAKE_CXX_FLAGS_RELEASE_OVERRIDE STRING : -DNDEBUG
+opt-set-cmake-var CMAKE_C_FLAGS STRING : -O2 -finline-functions -w
+opt-set-cmake-var CMAKE_C_FLAGS_RELEASE_OVERRIDE STRING : -DNDEBUG
+opt-set-cmake-var CMAKE_FORTRAN_FLAGS STRING : -O2 -finline-functions -w
```

```suggestion
opt-set-cmake-var CMAKE_CXX_FLAGS STRING : -O2 -finline-functions
opt-set-cmake-var CMAKE_C_FLAGS STRING : -O2 -finline-functions
opt-set-cmake-var CMAKE_FORTRAN_FLAGS STRING : -O2 -finline-functions
```

We shouldn't need to set the NDEBUG stuff, that was an old workaround in their build system I think.  Also no need to set `-w` (they don't want to see warnings from their TPL builds, so they disable it).  I think generally we (Trilinos) _want_ to see those warnings.